brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/platform/ToscaPlatform.java

```diff
@@ -0,0 +1,18 @@
+package io.cloudsoft.tosca.a4c.platform;
+
+import java.io.Closeable;
+
+import io.cloudsoft.tosca.a4c.brooklyn.ToscaApplication;
+
+public interface ToscaPlatform extends Closeable{
```

Comment:
In IntelliJ cmd+shift+enter completes the line (brackets, semi-colons, etc.) and will format for you.